ictv/libs/html.py

```diff
@@ -224,11 +224,12 @@
 XHTML support was contributed by Michael Haubenwallner.
 '''
 from __future__ import with_statement
+from cgitb import html
```

Comment:
It looks like this package will be deprecated in Python 3.11. Can you see if this can't be replaced by any Flask API ?